src/modules/playRoom/components/CountDown.js

```diff
@@ -1,11 +1,27 @@
-import React from 'react';
+import React, { useEffect, useState }  from 'react';
 
 const CountDown = () => {
+  const [countDown, setCountDown] = useState(59);
+
+  useEffect(() => {
```

Comment:
as I said earlier, we won't let the count down happens on front end side. Instead we should handle the event sent from server to update the clock value